### ISSUE TITLE
Performance issues after quitting out of xmr-stak

### ISSUE THREAD

**Issue by DanB91** [fireice-uk/xmr-stak#707]
I am mining with xmr-stak.  I typically have 2 instances opened, one for CPU  and one for GPU (I've read this can have better difficulty distribution in pools?)

My setup is:
 - Intel i5-7600k OC'd at 4ghz
 - NVIDIA GTX 1070 stock (driver from 12/5/2017, but previous drivers had the same issue)
 - NVIDIA GTX 670 stock driver from 12/5/2017, but previous drivers had the same issue)
 - Windows 10
 - xmr-stak 2.1.0 31a0784 (compiled from source)

Occasionally I quit out from these 2 instances of xmr-stak to play a game or do something else.  Once I do that, I notice that the mouse cursor occasionally skips when moving around the screen and games can become unplayable due to the choppiness.  Once I restart my computer, everything starts running smoothly again.  

So essentially, if I want to do something on my computer, I have to quit out of the miners and restart my computer.

I double check Task Manager and xmr-stak isn't running, nor is there any program taking up significant CPU usage.  Has anyone run into this issue?

Thanks!

-Daniel

**Comment by KeysorSozer** [comment]
Do you have monerod running at the same time? I noticed that running xmr-stack while also running monerod resulted in monerod eating all my RAM but not reporting it in task manager, could only tell from performance monitor due to it reserving the RAM but not actually using it. I would also get the same thing.

**Comment by DanB91** [comment]
@KeysorSozer Nah I don't (double checked task manager).  I haven't used the Monero GUI or monerod up in weeks.  I'll keep that in mind, though.  Thank you.

**Comment by semtex41** [comment]
I have seen this also. Both with version 2.0.0 and 2.2.0. 
The mouse cursor was moving visibly slower (logitech g502). 
Rebooting the machine appears to fix the issue.

**Comment by JerichoJones** [comment]
Can you try mining CPU Only then testing?
Rinse and repeat with the GPU.

**Comment by DanB91** [comment]
I think I might have figured it out.

I had did a git pull and compiled last night.  I noticed that the stuttering was even worse than before and windows itself became unusable after quitting out both the CPU and GPU miner.  But before I rebooted the computer, I was able to get into Performance Manager and noticed that system interrupts were high CPU usage (~35%).  I also noticed that an "APCent.exe" would occasionally show up as not responding.

Turns out, this belongs to the Gigabyte app center (which came with my motherboard).  After rebooting and then killing APCent.exe before starting mining, I seem to no longer have this stuttering issue.

It's important to note both my motherboard and graphics card are gigabyte.  I tried what @JerichoJones suggested to see if it's either the GPU, CPU or combo of the GPU/CPU miner screwing things up, but it seems I can only reproduce this issue if I let it run for a long amount of time.

I'm honestly fine with just making sure APCent.exe isn't running while running the miner (there is no need for me to have it running), and also I feel it might be more of a bug in APCent than the miner.  But, if you guys feel it would be of value, I can investigate further to see if its the CPU miner, the GPU miner, or a combo of both.
